SourceX/DiabloUI/credits.cpp

```diff
@@ -154,10 +148,10 @@ class LinesBuffer {
 			empty_ = true;
 	}
 
-	void push_back(CachedLine &&line)
+	void push_back(CachedLine line)

```

Comment:
@glebm this is the change that I suspect will have a negative effect on performance.

Reply:
`CachedLine` is small so shouldn't make a difference